src/components/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,267 @@
+
+    if (result.MESSAGE === 'SUCCESS') {
+      navigate('/cart');
+    }
+  };
+
+  const deleteHandler = e => {
+    const afterDelete = ordersList.filter(item => item.id !== ordersList.id);
+  };
+
+  if (Object.keys(detail).length !== 0) {
+    return (
+      <section className="rightPanel">
+        <header className="listHeader">
+          <h3>SHOP</h3>
+          <p>
+            <span>HOME ＜</span> SHOP
+          </p>
+        </header>
+        <div className="listContainer">
+          <article className="description">
+            <div className="productImages">
+              <div className="productThumbnail">
+                {imageSelect === '' ? (
+                  <img src={detail.img[0].img_url} alt="thumbnail" />
+                ) : (
+                  <img src={imageSelect} alt="thumbnail" />
+                )}
+              </div>
+              <div className="awaitingImages">
+                <img
+                  src={detail.img[0].img_url}
+                  alt="thumbnail"
+                  onClick={() => {
+                    setImageSelect(detail.img[0].img_url);
+                  }}
+                />
+                <img
+                  src={detail.img[1].img_url}
+                  alt="thumbnail"
+                  onClick={() => {
+                    setImageSelect(detail.img[1].img_url);
+                  }}
+                />
+              </div>
+            </div>
+
+            <div className="productDetails">
+              <div className="describe">
+                <h3 className="productTitle">{detail.name}</h3>
+                <div className="defaultPrice">
+                  {Math.floor(detail.price).toLocaleString('ko-KR')}원
+                </div>
+                <div className="productTaste">
+                  <p>
+                    Tasting Note
+                    {detail.taste.map(taste => (
+                      <span key={taste.taste_id}>{taste.taste_name}</span>
+                    ))}
+                  </p>
+                </div>
+              </div>
+
+              <form className="optionSelect">
+                <div className="productRoasting">
+                  <div className="indicator">
+                    <p>분쇄</p>
+                  </div>
+                  <div className="buttonDetail">
+                    <div className="choiceButtonWrapper">
+                      {detail.graind.map(grind => (
+                        <button
+                          className={`${
+                            Number(order.graind) === grind.graind_id
+                              ? 'highlighted'
+                              : ''
+                          }`}
+                          name="graind"
+                          key={grind.graind_id}
+                          value={grind.graind_id}
+                          onClick={e => {
+                            productOptionHandler(e);
+                          }}
```

Comment:
```suggestion
                          onClick={productOptionHandler}
```
꼭 event를 인자로 넘겨주지 않아도, 이미 onClick이라는 이벤트가 동작했을 때 실행하는 함수기 때문에
자동적으로 event가 넘어가게 됩니다.

Reply:
아하 감사합니다 바로 반영할 수 있도록 하겠습니다!